routes/auth.js

```diff
@@ -1,9 +1,9 @@
 const jwt = require("jsonwebtoken");
 const Joi = require("joi");
 const bcrypt = require("bcryptjs");
-const { User, validate } = require("../models/users");
-const mongoose = require("mongoose");
+const user = require("../controllers/user");
```

Comment:
Lets try to abstract most of the db logic so that no there only one place to do changes if required in future instead of multiple places